### ISSUE TITLE
Necrotizing ileitis

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6855]
**Preferred term label:**
Necrotizing ileitis

**Synonyms**
Necrotic ileitis

**Definition (free text, please give PubMed ID)**
Necrotizing inflammation of the ileum.
PMID: 11529377

**Parent term (use hpo.jax.org/app)**
Ileitis

**Comment by pnrobinson** [comment]
@johannaest  -- I think you probably mean Necrotizing enterocolitis HP:0033165 ??
https://hpo.jax.org/app/browse/term/HP:0033165
If this is not what you mean please reopen this term request and provide a definition that differeniates Necroitizing ileitis from Necrotizing enterocolitis -- there are not many reports but those I could find are very similar.

**Comment by johannaest** [comment]
Ileitis and enterocolitis are different things, enterocolitis refers to small and large intestine, while ileitis means only a certain part of the small intestine. It is true that necrotizing ileitis is rarely described, whereas necrotizing enterocolitis is a relatively common condition in preterm infants.
